update-semantic-forms.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Extending the build-forms script
+# (Not recommended but you can comment this part but it will result in duplicate if you run the command twice)
+echo "Building the forms..."
+bash ./build-forms.sh
+echo "Forms built successfully!"
+echo ""
+echo "---------------------------------------"
+echo ""
+
+# Required : Add your path where you store your projects this has to start with / and end with /
+path_folder="/path/to/yours/projects/"
```

Comment:
I would favour the scripts reeds an .env file where you spec the full path per app you want it propagated to.

Reply:
and fails when no env file is found.

Reply:
Thanks for the review, I'll look into that change :+1: